src/android/plugin.gradle

```diff
@@ -2,11 +2,18 @@ repositories {
     maven {
         url 'https://maven.localytics.com/public'
     }
+    maven {
```

Comment:
Since this is a open source repo. I wouldn't put merge the change to our private SDK repo.